examples/osm-vector-tiles.js

```diff
@@ -3,58 +3,60 @@ import TopoJSON from '../src/ol/format/TopoJSON.js';
 import VectorTileLayer from '../src/ol/layer/VectorTile.js';
 import VectorTileSource from '../src/ol/source/VectorTile.js';
 import View from '../src/ol/View.js';
-import {Fill, Stroke, Style} from '../src/ol/style.js';
 import {fromLonLat} from '../src/ol/proj.js';
 
 const key = 'uZNs91nMR-muUTP99MyBSg';
 
-const roadStyleCache = {};
-const roadColor = {
-  'major_road': '#776',
-  'minor_road': '#ccb',
-  'highway': '#f39',
-};
-const buildingStyle = new Style({
-  fill: new Fill({
-    color: '#666',
-    opacity: 0.4,
```

Comment:
The Fill constructor doesn't have an `opacity` option, so this had no effect.  In the updated style, I've set the opacity in the `fill-color` and `stroke-color`, so the rendered output looks different.